### ISSUE TITLE
BUILD_TESTS does nothing

### ISSUE THREAD

**Issue by orbea** [Grumbel/sdl-jstest#20]
In commit https://github.com/Grumbel/sdl-jstest/commit/5bc418e7f12b6da24992b414d1e06ce424d30e5c the appdata files were removed, but the `BUILD_TESTS` cmake option remained.

https://github.com/Grumbel/sdl-jstest/blob/a73cfcbd95ec3fb62e65340f1e3ec43648e50491/CMakeLists.txt#L52-L55

It now does nothing since the appdata files are gone, what was the intent here? Are different tests going to be created in the future? Should `BUILD_TESTS` be removed altogether? Should the appdata files be restored? I don't have a personal preference here.

**Comment by orbea** [comment]
@Grumbel I would really appreciate your input here.

**Comment by Apteryks** [comment]
Indeed, I also get: "No tests were found!!!" when running 'make check'.
